providers/postmark/src/lib/postmark.provider.ts

```diff
@@ -72,6 +72,8 @@ export class PostmarkEmailProvider implements IEmailProvider {
       HtmlBody: options.html,
       TextBody: options.html,
       Subject: options.subject,
+      Cc: options.cc ? options.cc[0] : undefined,
+      Bcc: options.bcc ? options.bcc[0] : undefined,
```

Comment:
Same question as for Mailgun. Doesn't Postmark take `cc` and `bcc` as list of emails?

Reply:
Not according to the interface from the postmark package.

Reply:
Same comment as for Mailgun then. 🙁 
https://github.com/novuhq/novu/pull/2763#discussion_r1105756766

Reply:
@scopsy should we mention this in the docs maybe?
ping @iampearceman and @jainpawan21

Reply:
Never mind found a way of doing it interfaces was not clear enough